app/services/scheduler.py

```
@@ -178,110 +176,158 @@ def _add_stats_cleanup_job(self):
+        except Exception as e:
+            logger.exception(f"Error adding disk space monitoring job: {e}")
 
 
 def check_disk_space_and_notify(path: MonitoredPath, db: Session):
     """Check disk space for all cold storage locations and send notifications if low."""
-    # TODO: Make this threshold configurable
-    low_space_threshold_percent = 90.0
-
     for location in path.storage_locations:
         try:
             total, used, free = shutil.disk_usage(location.path)
```

Ruff (0.14.13)

Unpacked variable `used` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)